src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1229,6 +1230,12 @@ object FunctionRegistry {
     "b" -> "the base.",
     "x" -> "the exponent.")
 
+  register("gamma", (x: Double) => Gamma.gamma(x),
+    """
+    Returns the value of ``Gamma(x)``.
```

Comment:
To avoid confusion with functions associated with the gamma distribution, I'd write:

Returns the value of the `gamma function <https://en.wikipedia.org/wiki/Gamma_function>`__ at ``x``.